src/lib.rs

```diff
@@ -273,6 +267,7 @@ pub struct Field {
     pub bit_range: BitRange,
     pub access: Option<Access>,
     pub enumerated_values: Vec<EnumeratedValues>,
+    pub write_constraint: Option<WriteConstraint>,
```

Comment:
We should add a private field to reserve the right to extend this structure without raising a minor version in the future.